life/updated/02-life.adoc

```diff
@@ -943,169 +943,171 @@ insomuch that I could hardly go on my feet,
-what can you make of that?`" I said, "`You are out of the order of the Gospel;
-for it is said, the church may exercise one by one;
-and you do not as you would be done by.`"
-And further I told him, "`That this was not his place to abide here preaching,
-and burdening the souls of the innocent; but your place is to return home into the north,
+what can you make of that?`" I said, "`You are out of the order of the Gospel,
+for it is said, '`the church may exercise one by one;`'
+and you do not do as you would have done to you.`"
+And further I told him, "`This is not your place to abide here preaching,
+and burdening the souls of the innocent. Your place is to return home into the North,
 and be reconciled to your brethren, before you go to offer your gift.`"
-Many great swelling words proceeded from him, and his three friends who were with him;
+Many great and lofty words proceeded from him, and his three friends who were with him,
 and they went away sorely displeased.
 
 Their rage increased towards me and many faithful
-Friends who had sat under their dead ministry;
-but mostly against me, for discharging my duty,
-in obedience to what the Lord required of me, and committed to my charge,
-concerning that spirit which for some time endeavored to lord it over God`'s heritage;
+Friends who had sat under their dead ministry,
+but mostly against me for discharging my duty
+in obedience to what the Lord required of me and committed to my charge,
+concerning that spirit which for some time endeavored to rule over God`'s heritage;
 which made many sensible ones go bowed down many a time.
 My soul is a living witness, with many more, of what I have here declared,
-which is but little of their persecution towards me, in consideration of what follows,
-for the Lord was pleased to continue my exercise in that city,
+which is but little of their persecution towards me in consideration of what follows,
+for the Lord was pleased to continue my exercise in that city
 where John Story abode much of his time.
-Several more of that spirit oftentimes frequented there,
+Several more of that dividing spirit oftentimes frequented there,
 and the Lord was pleased to make me so sensible of them,
 that in the night season I had many a sore travail of spirit,
 when I knew not of them by information from any one.
 Then did I cry unto the Lord in secret,
 "`What shall I do to go through such hard things? Oh, that I may be excused,
-or that you will be pleased to keep me in silence this day;
-then should I be very willing to go to meeting to wait upon you,
-and to sit under the shadow of your wing with great delight,
-where your fruit will be pleasant to my taste.`"
-Then would come up before me,
+or that You will be pleased to keep me in silence this day.
+Then should I be very willing to go to meeting to wait upon You,
+and to sit under the shadow of Your wing with great delight,
+where Your fruit will be pleasant to my taste.`"
+Then would come up before me
```

Comment:
Maybe change the order here: Then the covenant would come up before me that I made....